Add SLES 12 SP2 and OpenSUSE Leap 42.2 guests

morecache opened the pull request.

xutian commented on shared/unattended/OpenSUSE-Leap-42.2.xml:
```diff
@@ -0,0 +1,967 @@
+      <rule>
+        <name>eth0</name>
+        <rule>ATTR{address}</rule>
+        <value>52:54:00:7d:73:7f</value>
```
Better to not fix MAC here.

morecache replied:
the mac part was removed.

morecache commented:
Ping

xutian commented:
@morecache   will test it, sorry for response so slowly.

xutian commented:
@morecache   installation timeout in my host.
[root@amd-5400b-4-4 avocado-vt]# ./scripts/scan_results.py 
CaseName                                 Status          StartTime       EndTime         Result     TimeElapsed
'1-smp_1.2048m.repeat1.pre_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.OpenSUSE.42.2.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_native' Finshed 2017-04-10 03:01:52 2017-04-10 04:24:13 ERROR 4940.348177
'2-smp_1.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.OpenSUSE.42.2.x86_64.io-github-autotest-qemu.boot' Finshed 2017-04-10 04:24:13 2017-04-10 04:28:42 ERROR 268.942775
Summary: 
Finshed=2, ERROR=2
[root@amd-5400b-4-4 avocado-vt]#

morecache commented:
@xutian set up 'Turn Off the Machine after the First Stage' in OpenSUSE-Leap-42.2.xml
It notifies avocado that the installation is done, avoid the vm installation infinite loop.

morecache commented:
Ping

xutian commented on shared/unattended/OpenSUSE-Leap-42.2.xml:
```diff
@@ -0,0 +1,961 @@
+      <package>dhcp-tools</package>
+      <package>sg3_utils</package>
+      <package>autoyast2-installation</package>
+      <package>autoyast2-installation</package>
```
looks duplicate lines

morecache replied:
ok, will drop them.

xutian commented on shared/unattended/OpenSUSE-Leap-42.2.xml:
```diff
@@ -0,0 +1,961 @@
+    <device_map config:type="list">
+      <device_map_entry>
+        <firmware>hd0</firmware>
+        <linux>/dev/vda</linux>
```
I'm not sure that is it works for virtio-scsi disk.

morecache replied:
Sorry for the delay, I've been keeping busy recently.

Good catch.
BTW, There is one more section which blocks the installation with virtio-scsi.
I will fix them later, Thanks for your review.

xutian commented:
@morecache  ping

morecache commented:
- Removed the duplicated lines.
- Fixed the virtio_scsi installation issue.
- Add post-script section to modify sshd_config to set PasswordAuthentication to enable tunneled clear text passwords.

morecache commented:
@xutian
I made some changes according to your review, But I used 'push --force' to update the repo, so some of your review gone, sorry about that.

morecache commented:
@xutian ping

luckyh commented:
@morecache the configuration file of `SLES.12.2.x86_64` has already been merged by https://github.com/avocado-framework/avocado-vt/pull/1028, so please update this PR with dropping it. 

And also, please provide the test result of `OpenSUSE.42.2.x86_64`.

Thanks!
